addons/project_purchase/models/project.py

```diff
@@ -1,14 +1,55 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-from odoo import fields, models
+from odoo import api, fields, models, _
 
 
 class Project(models.Model):
     _inherit = "project.project"
 
-    purchase_order_count = fields.Integer(related="analytic_account_id.purchase_order_count")
+    purchase_orders_count = fields.Integer('# Purchase Orders', compute='_compute_purchase_orders_count')
```

Comment:
**Detail**

Just so you know, we could keep the field as it was and fix the compute in `purchase`